ci-operator/step-registry/baremetalds/metallb-e2e/test/baremetalds-metallb-e2e-test-commands.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ baremetalds metallb e2e test command ************"
+
```

Comment:
If you have some relevant artifacts usefl for troubleshooting then they could be published in the job (`$ARTIFACTS_DIR`), see https://github.com/openshift/release/blob/bad39db84adafcad033a7060d7164eecb73815cf/ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh#L13 for example

Reply:
Thank you for the advice, currently there is no info file in metallb, so it could be future work, I guess.